e2e/tests/components/radiobuttons/radiobuttons.po.spec.ts

```diff
@@ -1,13 +1,11 @@
-import { browser, element, by } from 'protractor';
+import { browser, element, by, ElementFinder } from 'protractor';
 
 export class RadioButtonsPage {
         
     getPage(): void {
         browser.get('/radiobuttons');
     }
     
-    titleText = browser.getTitle();
-    
     radiobutton1 = element(by.id('radiobutton1'));

```

Comment:
Getting warnings on this page:

![image](https://user-images.githubusercontent.com/20795331/28207393-7b256018-6881-11e7-91d6-3e650e446798.png)